Factory/DiactorosFactory.php

```diff
@@ -86,6 +86,10 @@ private function getFiles(array $uploadedFiles)
         $files = array();
 
         foreach ($uploadedFiles as $key => $value) {
+            if ($value === null) {
```

Comment:
I've always been curious why the Symfony source prefers `null` to come first. What's the reasoning?